base-v3/src/main/java/com/adyen/checkout/base/util/PaymentMethodTypes.java

```diff
@@ -35,7 +35,6 @@
     public static final String SCHEME = "scheme";
     public static final String GOOGLE_PAY = "paywithgoogle";
     public static final String SEPA = "sepadirectdebit";
-    public static final String AFTER_PAY = "afterpay_default";
```

Comment:
Let's keep this and add it to the list of Unsupported ones.

Reply:
👍